src/Indodana.php

```diff
@@ -73,17 +71,19 @@ public function checkTransactionStatus(array $input = []) {
     return $responseJson;
   }
 
-  public function getInstallmentOptions(array $input = []) {
+  public function getInstallmentOptions(array $input = [])
+  {
     $url = $this->urlPath('/merchant/v1/payment_calculation');
     $header = $this->getDefaultHeader();
     $body = $input;
 
     $responseJson = IndodanaHttpClient::post($url, $header, $body);
 
-    return $responseJson['payments'];
+    return $responseJson;
```

Comment:
Is that okay to change the return value of this function ?
Will it broke another plugin that already use this function ?

Reply:
I already changed the plugin

Reply:
okay cool !